sapientml/params.py

```diff
@@ -379,7 +379,6 @@ def __init__(
             filename = "training." + ("pkl" if save_datasets_format == "pickle" else "csv")
             self.training_data_path = str(self.output_dir / filename)
             save_file(self.training_dataframe, self.training_data_path, csv_encoding, csv_delimiter)
-            self.training_data_path = str(Path(self.training_data_path).relative_to(self.output_dir))
```

Comment:
Why did you remove this line?

Reply:
In line 380, "self.training_data_path" is the absolute path, but in line 382, it is changed to only the file name, and failed to load the file when reloading.

Reply:
Thank you. No problem.